build.sbt

```diff
@@ -105,18 +105,23 @@ lazy val validation = (project in file("validation"))
 
 lazy val panel = (project in file("panel"))
   .settings(hmdaBuildSettings: _*)
+  .settings(Revolver.settings:_*)
   .settings(
-    assemblyMergeStrategy in assembly := {
-      case "application.conf" => MergeStrategy.concat
-      case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
-      case x =>
-        val oldStrategy = (assemblyMergeStrategy in assembly).value
-        oldStrategy(x)
-    },
-    libraryDependencies ++= akkaPersistenceDeps ++ httpDeps
+    Seq(
+      assemblyJarName in assembly := {s"${name.value}.jar"},
+      mainClass in assembly := Some("hmda.panel.PanelCsvLoader"),
+      assemblyMergeStrategy in assembly := {
+        case "application.conf" => MergeStrategy.concat
+        case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
+        case x =>
+          val oldStrategy = (assemblyMergeStrategy in assembly).value
+          oldStrategy(x)
+        },
+      libraryDependencies ++= httpDeps,
+      fork in run := true
```

Comment:
It forks the JVM in which the panel loader runs, allowing it to exit and return to the `sbt`, rather than exiting to the terminal.  This is only for usability reasons; it seemed like killing `sbt` wasn't good practice.